fre/activites.md

```diff
@@ -133,36 +133,35 @@ L'Archivage est le processus qui consiste à déplacer des données ou d'autres
 
 Organisation
 ------------
-L'Organisation concerne l'arrangement des objets (matériaux de recherche, jeux de données, images, etc.) d'une façon qui facilite d'autres activités de recherche. Elle peut également comprendre des activités qui favorisent la découverte, comme la création de métadonnées et leur amélioration.
+L'Organisation concerne l'arrangement des objets (matériaux de recherche, jeux de données, images, etc.) d'une façon qui facilite d'autres activités de recherche. Elle peut également comprendre des activités qui favorisent la découverte, comme la création de métadonnées ("Annotation") et leur amélioration ("Enrichissement").
 
 Conservation
 ------------
-La Conservation est l'application de stratégies, d'activités et de technologies spécifiques en vue de garantir qu'un contenu numérique sera correctement rendu au fil du temps. Elle facilite le réemploi des données, des objets et des ressources liés à la recherche et peut comprendre des activités en rapport avec la durabilité et l'interopérabilité.
-Les techniques associées sont, notamment, les suivantes : Bit Stream Preservation, Supports pérennes, Émulation, Liaison des métadonnées, Migration, Réplication, Conservation de technologie, Versionnage, recours à un Open Archival Information System et à des standards rendant possible l'interopérabilité.
+La Conservation est l'application de stratégies, d'activités et de technologies spécifiques en vue de garantir qu'un contenu numérique sera correctement rendu au fil du temps. Elle facilite le réemploi des données, des objets et des ressources liés à la recherche et peut comprendre des activités en rapport avec la durabilité et l'interopérabilité. Les techniques associées sont, notamment, les suivantes :" Bit Stream Preservation", "Supports pérennes", "Émulation", Liaison des métadonnées, "Migration", "Réplication", "Conservation de technologie", "Versionnage", recours à un "Open Archival Information System" et à des standards rendant possible l'interopérabilité.
 
 7 – Dissémination
 =================
-La Dissémination est l'activité qui consiste à rendre disponibles des objets d'enquête, des résultats de recherche ou des logiciels et des services à l'intention de collègues chercheurs ou d'un public plus large, de diverses façons plus ou moins formelles. Elle exige un stockage et peut comprendre une publication et un partage de données recourant à diverses méthodes et techniques, dont l'application des principes des données liées ouvertes.
+La Dissémination est l'activité qui consiste à rendre disponibles des objets d'enquête, des résultats de recherche ou des logiciels et des services à l'intention de collègues chercheurs ou d'un public plus large, de diverses façons plus ou moins formelles. Elle exige un stockage et peut comprendre une publication et un partage de données recourant à diverses méthodes et techniques, dont la mise en place de données liées ouvertes.
```

Comment:
"la mise en place de données liées ouvertes" > "la mise en œuvre de données liées ouvertes"

Cela simplifie un peu, mais reste opaque, je trouve. L'expression "application of linked open data" était un peu curieuse. Du point de vue du sens, je préfère encore "l'application des principes des données liées ouvertes". Arguments ?